quay/quay
# ui: implementing creation of custom git trigger (PROJQUAY-6299)

Implements the build trigger creation wizard. 

**How to test**
1. Enable the following configs `FEATURE_BUILD_SUPPORT` and `FEATURE_UI_V2_BUILDS`
2. Go to a repository -> Builds -> Create build trigger
3. Enter the required information and the build should be successfully created
